Source/Fuse.Elements/Caching/ElementAtlas.uno

```diff
@@ -167,8 +159,12 @@ namespace Fuse.Elements
 			_framebuffer.Unpin();
 		}
 
-		void FillFramebuffer(DrawContext dc, bool drawAll, Rect scissorRectInClipSpace)
+		void FillFramebuffer(DrawContext dc, framebuffer fb, bool drawAll, Rect scissorRectInClipSpace)
 		{
+			// Changing the framebuffer is expensive. This loop actually doesn't draw anything in
+			// common case. Don't push the render target unless we actually have to render something.
```

Comment:
@Duckers: Yeah, I think so. At least for now. Maybe it would be an idea to fix the underlying issue, but I don't have a great strategy for this that doesn't involve linear scanning yet. Perhaps some annotated KD-tree or something, but that's a bigger project.